hydro_cli/src/core/ssh.rs

```diff
@@ -188,13 +188,19 @@ impl<T: LaunchedSSHHost> LaunchedHost for T {
         let port = addr.port();
 
         tokio::spawn(async move {
+            #[allow(clippy::never_loop)]
             while let Ok((mut local_stream, _)) = local_port.accept().await {
                 let mut channel = session
                     .channel_direct_tcpip(&internal_ip, port, None)
                     .await
                     .unwrap();
                 let _ = tokio::io::copy_bidirectional(&mut local_stream, &mut channel).await;
+                break;
```

Comment:
Is this temporary? Or should this be an `if let` instead of `while let`

Reply:
Temporary, the true correct logic should be a while loop but doing so results in resource leaks, since we don't know when the forwarded port is no longer needed. So once I figure out a good solution for the TODO, I will be able to remove the `break`.